html/pfappserver/root/config/profile/files.tt

```diff
@@ -206,6 +206,7 @@ td a.btn { margin-left: 5px; }
       </table>
 
         <a class="btn btn-danger call-modal" data-modal="revertAll" href="[% c.uri_for(c.controller('Config::Profile').action_for('revert_all'),[id]) %]" ><i class="icon-refresh"></i> [% l('Revert All Files')  %]</a>
+    </div>
```

Comment:
Fixed

James Rouzier
jrouzier@inverse.ca :: +1.514.447.4918 (x115)  ::  http://www.inverse.ca
Inverse inc. :: Leaders behind SOGo (http://www.sogo.nu) and PacketFence (http://www.packetfence.org)

On 2016-03-18 2:37 PM, Julien Semaan wrote:
>
> In html/pfappserver/root/config/profile/files.tt 
> <https://github.com/inverse-inc/packetfence/pull/1208#discussion_r56703089>:
>
> > @@ -206,6 +206,7 @@ td a.btn { margin-left: 5px; }
> >        </table>
> >
> >          <a class="btn btn-danger call-modal" data-modal="revertAll" href="[% c.uri_for(c.controller('Config::Profile').action_for('revert_all'),[id]) %]" ><i class="icon-refresh"></i> [% l('Revert All Files')  %]</a>
> > +    </div>
>
> You deleted the opening div as well in your previous commit....
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/inverse-inc/packetfence/pull/1208/files/9bbabbacbef11f643c2aedbfb58b5909ea25a2ac..6a07e29f64118352dee06019ce13a01d9be8d584#r56703089>
>